### ISSUE TITLE
upgrade lime-icons8

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#707]
many icons are missing compared to their online version, and that limits our choices.

**Comment by magnusfagerlund** [comment]
I have looked in to this and apparently there is no other way than to manually add the icons missing (which is 3500) and download the collection. See response from Icons8 chat.

![icons 8](https://user-images.githubusercontent.com/7524499/79863263-bb315080-83d7-11ea-9e29-fd269ed4620c.png)

I have started the work to add the missing icons to my "collection" to be able to download. WIll do it whn I have more time.

**Comment by adrianschmidt** [comment]
@magnusfagerlund It's probably reasonably simple to make a script to do that, and then we can use that same script in the future as well.

**Comment by magnusfagerlund** [comment]
@adrianschmidt Yeah, that would be a much better solution of course. Dont know how build that script, but the API docs are found here; https://icons8.github.io/icons8-docs/#Icons%20API

**Comment by adrianschmidt** [comment]
Oh, they even have an API? That's great! I was just going to do something that navigates the website and clicks the *Add to Collection* button for you, but if we can get the icons via their API then that's much much better!

**Comment by magnusfagerlund** [comment]
Yeah, but it turns out that we don't have the API license so we wont be able to use that.

**Comment by adrianschmidt** [comment]
Still probably worth trying to automate the process of browsing and clicking the add-button…

**Comment by magnusfagerlund** [comment]
Yeah definitely... I would say it would take an hour or two to manually just clicking the button...

**Comment by Kiarokh** [comment]
If we're hosting icons ourselves, I can design, and create SVGs and add them to our collection too (in occasional cases, for example if we don't find what we need even if their updated icon sets). 

Anyhow, I'm don't know why we must host all the icons. Is it because of on-premise offline use like the fonts issue?

**Comment by adrianschmidt** [comment]
With icons8, I think loading the icons straight from them is charged per-request, which is probably not a risk we'd be all that happy to subject ourselves to…

**Comment by adrianschmidt** [comment]
Two significant benefits we'd get from *not* hosting ourselves are: 
- we'd probably be *much* less likely to accidentally violate the license agreement
- we'd get a much simpler install configuration for lime-elements, since that would no longer depend on a non-opensource package

**Comment by magnusfagerlund** [comment]
> Icons8 API allows users to search and obtain our icons.
> 
> Our starter API plan is $100/month - it includes up to 100 000 requests per month. Every 100 000 requests add $100 more: $200/month includes up to 200 000 requests per month, $300/month - up to 300 000 requests and so on. Contact us to get started.
>

**Comment by Kiarokh** [comment]
Wow, seems like a good business model... hmmmm.... Who's up for a startup with me?I make the icons, you do the math :D

**Comment by jensgustafsson** [comment]
When I read the comments here I have a hard time understanding what we actually decided to do. 

Manually clicking and downloading is not an option imho. What if we but their API plan? 100 000 requests should be enough to download 3500 icons 🤔

**Comment by adrianschmidt** [comment]
We never did anything about it. If the API-plan allows for downloading the icons and then keeping them forever and hosting them, then that's definitely one way to go about getting the icons. I wouldn't assume that's allowed without checking though. Although, if that isn't allowed, but the plan we already have give us permission to download and host not just the icons we already have, but any new icons as well, then I guess the combination might mean we can download them via the API and then host them ourselves…? I think we should clear that up with their support before paying for an API-plan, even if just for one month.

Are the login credentials in passwordstate? If I can log in, it's probably not too difficult to make a script that navigates through the site and adds all the icons to the download basket.

**Comment by jensgustafsson** [comment]
> We never did anything about it. If the API-plan allows for downloading the icons and then keeping them forever and hosting them, then that's definitely one way to go about getting the icons. I wouldn't assume that's allowed without checking though. Although, if that isn't allowed, but the plan we already have give us permission to download and host not just the icons we already have, but any new icons as well, then I guess the combination might mean we can download them via the API and then host them ourselves…? I think we should clear that up with their support before paying for an API-plan, even if just for one month.
> 
> Are the login credentials in passwordstate? If I can log in, it's probably not too difficult to make a script that navigates through the site and adds all the icons to the download basket. (If I remember correctly, there's an add-to-basket button when not logged in too, but I don't want to spend the time hacking together a script, only to realise it needs to be completely different when logged in…)

Yes, the credentials are in pwstate, but under Marketing so we need to ask them for it.
